tests/task/task_runner/test_standard_task_runner.py

```diff
@@ -121,11 +121,16 @@ def test_on_kill(self):
         runner = StandardTaskRunner(job1)
         runner.start()
 
-        # give the task some time to startup
-        time.sleep(3)
-
+        # Give the task some time to startup
+        time.sleep(10)
         runner.terminate()
 
+        # Wait some time for the result
+        for _ in range(5):
+            if os.path.exists(path):
+                break
+            time.sleep(8)
```

Comment:
8 seconds seems like a lot, i would recommend increase the range count and decrease the sleep interval here.

Reply:
Agree, fixed